assets/js/editor/editor.js

```diff
@@ -66,7 +66,9 @@ class ConvertToBlocksEditorSupport {
 						return null;
 					}
 
-					client.save();
+					setTimeout(() => {
+						client.save();
+					}, config.agent.save_delay);
```

Comment:
@Sidsector9 Lets add a JS fallback to 0 in case the BE doesn't send this var. Sometimes this happens on large migrations when the BE stalls. If `save_delay` is 0, then call save directly instead of using the `setTimeout`.